src/highlight/HighlightAnnotations.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React, { useCallback } from 'react';
```

Comment:
We use `React.useState` and such elsewhere, so it seems prudent to continue to use the namespaced style here.